app/api/sul_bib/publications_api.rb

```diff
@@ -120,17 +100,15 @@ def original_source
       requires :id
     end
     get ':id' do
-      pub = publication_find(params[:id])
-      pub.pub_hash
+      publication_find(params[:id]).pub_hash
     end
 
     desc 'DELETE - MARK A RECORD AS OBSOLETE'
     params do
       requires :id
     end
     delete ':id' do
-      pub = publication_find(params[:id])
-      pub.delete!
+      publication_find(params[:id]).delete!
     end
```

Comment:
:+1: on the chain change;  I understand this doesn't change the existing behavior, so it's fine.  Looking at it prompts a thought about whether HTTP response codes should be different depending on whether the record is already deleted (i.e. it's a no-op) or this call does in fact delete it.  If those HTTP response codes should be different, we might want a backlog issue to look into that so this API is better RESTful citizen.